src/app/shared/helpers/state.ts

```diff
@@ -2,7 +2,14 @@
 export type FetchState<T> = Ready<T>|Fetching<T>|FetchError;
 export type Ready<T> = Readonly<{ tag: 'ready', isReady: true, data: T, isFetching: false, isError: false, error?: undefined }>;
 export type Fetching<T> = Readonly<{ tag: 'fetching', isReady: false, data?: T, isFetching: true, isError: false, error?: undefined }>;
-export type FetchError = Readonly<{ tag: 'error', isReady: false, data?: undefined, isFetching: false, isError: true, error: Error }>;
+export type FetchError = Readonly<{
+  tag: 'error',
+  isReady: false,
+  data?: undefined,
+  isFetching: false,
+  isError: true,
+  error: Error & { status?: number },
```

Comment:
To me it's all good, for this part I'd like @smadbe opinion

Reply:
Hum I don't like the idea of polluting the error with possible more-specific errors. It's clear there are improvement to be done (for instance, having a fetch state with a type of data and a type of error).
I think the most pragmatic approach would be to keep the previous version of the `state.ts` file and to use in template `$any(state.error).status !== 403` so that `state.error` is converted to `any` to try to fetch more specific error properties.

Reply:
Fixed